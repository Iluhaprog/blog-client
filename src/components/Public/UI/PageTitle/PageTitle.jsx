import styled from 'styled-components';

const PageTitle = styled.h1`
  max-width: fit-content;
  margin: 0 auto;
  font-size: 30px;
  font-family: var(--public-font-2);
  font-weight: 700;
  display: flex;
  flex-direction: column;
  align-items: center;
  
  &:after {
    content: '';
    width: 80%;
    height: 5px;
    background-color: #000;
    display: block;
    margin-top: 10px;
  }
`;

export {PageTitle};
