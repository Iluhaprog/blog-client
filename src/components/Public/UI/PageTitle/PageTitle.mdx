---
name: PageTitle
route: /page-title
---

import {Playground} from 'docz';
import {PageTitle} from './PageTitle';

## Basic usage
<Playground>
  <PageTitle>Page Title</PageTitle>
</Playground>
