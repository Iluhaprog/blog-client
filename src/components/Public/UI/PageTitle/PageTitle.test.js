import {render} from '@testing-library/react';
import {PageTitle} from './PageTitle';

test('Should render PageTitle component', () => {
  const {asFragment} = render(
    <PageTitle>
      TEST TITLE
    </PageTitle>
  );
  expect(asFragment()).toMatchSnapshot();
});
