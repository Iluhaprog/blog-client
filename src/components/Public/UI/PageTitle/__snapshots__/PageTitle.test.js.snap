// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render PageTitle component 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdfBwQ cZloQo"
  >
    TEST TITLE
  </h1>
</DocumentFragment>
`;
